code/game/objects/items/weapons/melee/misc.dm

```diff
@@ -22,8 +22,8 @@
 
 
 /obj/item/weapon/melee/rapier
-	name = "captain's rapier"
-	desc = "An elegant weapon, for a more civilized age."
+	name = "officer's sabre"
+	desc = "An elegant weapon, it's monomolecular edge is capable of cutting through flesh and bone with ease."
```

Comment:
its*

My palsy English makes be believe an apostrophe denotes ownership when using a noun however it's always means it is. English is evil and we should all speak Clanger.